tests/CommonExtractorTest.php

```diff
@@ -725,6 +725,36 @@ public function testIncrementalFetchingByAutoIncrement(): void
         $this->assertEquals(2, $newResult['imported']['rows']);
     }
 
+    public function testIncrementalMaxNumberValue(): void
+    {
+        $config = $this->getIncrementalFetchingConfig();
+        $config['parameters']['incrementalFetchingColumn'] = 'number';
+        $this->createAutoIncrementAndTimestampTable();
+
+        $result = ($this->getApp($config))->run();
+
+        $this->assertEquals('success', $result['status']);
+        $this->assertEquals(
+            [
+                'outputTable' => 'in.c-main.auto-increment-timestamp',
+                'rows' => 2,
+            ],
+            $result['imported']
+        );
+
+        $this->db->exec(
+            'INSERT INTO auto_increment_timestamp (`name`, `number`)'.
+            ' VALUES (\'charles\', 20.23486237628), (\'william\', 21.2863763287638276)'
+        );
+
+        $newResult = ($this->getApp($config, $result['state']))->run();
+
+        $this->assertArrayHasKey('state', $newResult);
+        $this->assertArrayHasKey('lastFetchedRow', $newResult['state']);
+        $this->assertEquals('21.286376953125', $newResult['state']['lastFetchedRow']);
```

Comment:
@ondrajodas vis proc je MAX `21.286376953125` kdyz tam mame `21.2863763287638276`?

Reply:
shit, toho jsem si nevšiml - kontroloval jsem jen pár začátečních čísel, ale vím :( 
to je bohužel vlastnost FLOAT datového typu že to takhle "upraví" - přehodil jsem to na DECIMAL
https://github.com/keboola/db-extractor-common/pull/136/commits/a58e90d9478029fda44169d2ca9bf5a25a966fb7

 a už by to mělo být přesné

Reply:
ok.  neni to velkej vec, FLOAT ma taky fungovat. ale incremental fetching na float nebude skoro nikdo nepouzivat a pokud to budou zkusit je to jasne ze to `neni exact`.  tak cajk